parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -31,6 +31,7 @@ spec:
         - "--service-account-key-file=/etc/kubernetes/certs/apiserver.key"
         - "--storage-backend=etcd2"
         - "--v=4"
+        - "--authorization-mode=RBAC"
```

Comment:
Can you please conditionalize this and bubble it up into the API model? User should explicitly turn RBAC on. I can raise a PR against your branch with the change